apps/web/src/pages/ProfileAndApplicationsPage/components/TrackApplications/TrackApplicationsCard.tsx

```diff
@@ -1,122 +1,218 @@
+    : getStatusPillInfo(application.status, intl);
+
+  const applicationDateInfo = getApplicationDateInfo(application, intl);
+  const { user } = useAuthorization();
+  const applicationTitle = getFullPoolTitleHtml(intl, application.pool);
   return (
     <div
-      data-h2-background="base(foreground)"
-      data-h2-shadow="base(medium)"
-      data-h2-radius="base(rounded)"
-      data-h2-padding="base(x1)"
+      data-h2-border-left="base(x.5 solid primary)"
+      data-h2-padding="base(x1 x1 x.5 x1)"
+      data-h2-shadow="base(larger)"
       data-h2-margin="base(0, 0, x.5, 0)"
+      data-h2-radius="base(0px rounded rounded 0px)"
     >
       <div
         data-h2-display="base(flex)"
-        data-h2-gap="base(0, x.5)"
-        data-h2-justify-content="base(space-between)"
+        data-h2-flex-direction="base(column) l-tablet(row)"
+        data-h2-gap="base(x1)"
       >
-        <Heading
-          level={headingLevel}
-          size="h6"
-          data-h2-margin="base(0, 0, x1, 0)"
-          data-h2-flex-grow="base(1)"
-        >
-          {getFullPoolTitleHtml(intl, application.pool)}
-        </Heading>
-      </div>
-      <div data-h2-margin="base(0, 0, x1, 0)">
-        <TrackApplicationsStatus application={application} />
-      </div>
-      <div>
-        {skills.length > 0 ? (
-          <div
-            data-h2-display="base(flex)"
-            data-h2-flex-wrap="base(wrap)"
-            data-h2-gap="base(x.25, x.125)"
+        <div data-h2-flex-grow="l-tablet(1)">
+          <Heading
+            level={headingLevel}
+            size="h6"
+            data-h2-margin="base(0, 0, x.5, 0)"
+            data-h2-flex-grow="base(1)"
           >
-            <p data-h2-margin="base(x.5, x.5, x.25, 0)">
-              {intl.formatMessage({
-                defaultMessage: "Assessed skills:",
-                id: "7NnNrW",
-              })}
-            </p>
-            <Chips>
-              {skills.map((skill: Skill) => (
-                <Chip
-                  key={skill.id}
-                  color="secondary"
-                  mode="outline"
-                  label={skill.name[locale] ?? ""}
-                />
-              ))}
-            </Chips>
+            {applicationTitle}
+          </Heading>
+          <div data-h2-display="base:children[>span](block) l-tablet:children[>span](inline-block)">
+            <span data-h2-color="base(primary.darker)">
+              {getRecruitmentType(application.pool.publishingGroup, intl)}
+            </span>
+            <span
+              data-h2-display="base(none) l-tablet(inline-block)"
+              data-h2-color="base(black.light)"
+              data-h2-margin="base(0, x.5)"
+            >
+              •
+            </span>
+            <span data-h2-color="base(black.light)">
+              {applicationDateInfo.message}
+              <span data-h2-color={applicationDateInfo.color}>
+                {" "}
```

Comment:
Any reason for the empty space?

Reply:
ugh.. I've an auto format which saves like this..  Idk why.. will remove